lib/Text/Xslate/Symbol.pm

```diff
@@ -45,7 +47,7 @@ has value => (
         return;
     },
 #    default => sub{
-#        if(!defined $_[0]) { #XXX: Mouse::XS's bug
+#        if(!defined $_[0]) { #XXX: Moo::XS's bug
```

Comment:
There is no such thing as Moo::XS - this is a bad search-replace.

Reply:
Okie. commented out default removed.